.github/workflows/build.yml

```diff
@@ -96,11 +96,15 @@ jobs:
       run: dotnet tool restore
 
     - name: Check format
-      run: dotnet fantomas --check src
+      run: dotnet fantomas --check build.fsx src
       env:
         DOTNET_ROLL_FORWARD: LatestMajor
         DOTNET_ROLL_FORWARD_TO_PRERELEASE: 1
 
+    # Ensure the scaffolding code can still add items to the existing code.
+    - name: EnsureCanScaffoldCodeFix
```

Comment:
@dawedawe  / @nojaf the codefix isn't working on this merge for some reason - can you take a look when you get a chance please?